cloud/amazon/ec2_vpc_nat_gateway_facts.py

```diff
@@ -147,7 +147,7 @@ def main():
 
     results = get_nat_gateways(connection, module)
 
-    module.exit_json(result=camel_dict_to_snake_dict(results))
+    module.exit_json(result=results)
```

Comment:
Rather than this, i would do:

` snaked_results = []
    for gateway in results:
        snaked_results.append(camel_dict_to_snake_dict(gateway))


    module.exit_json(nat_gateways=snaked_results)`